x-pack/docs/en/watcher/transform/search.asciidoc

```diff
@@ -56,10 +56,6 @@ The following table lists all available settings for the search transform:
 
 | `request.indices`                             | no        | all indices       | One or more indices to search on.
 
-| `request.types`                               | no        | all types         | One or more document types to search on (may be a
```

Comment:
I think this doc change snuck into this PR? It seems completely separate from this change

Reply:
Hah you fixed it right as I commented :)

Reply:
your fast!  yeah, I was on the wrong branch, that commit reverted.